### DISCUSSION TITLE
Unable to find Feed the Beast modpack ID

### DISCUSSION THREAD

**Discussion by davidfifth** [itzg/docker-minecraft-server#1277] [Q&A] [answered]
Hello all,

In the README, I'm trying to figure out the modpack ID mentioned in this line:

> - `FTB_MODPACK_ID`: **required**, the numerical ID of the modpack to install. The ID can be located by finding the modpack (https://www.feed-the-beast.com/modpack) and using the "ID" displayed next to the name

I don't see an ID anywhere there. I've been looking for an hour on how to find this, and I can't even find the "31" ID mentioned in the Direwolf20 pack example. I'm trying to find the modpack ID for FTB Infinity Evolved 1.7.10

**Comment by itzg** [comment, marked as answer]
Oh yeah, I don't see it on their pages anymore. That's frustrating since it's their own tool the script using to process the requested ID.

**Reply by davidfifth** [reply]
I suppose it's still possible to use a Feed the Beast pack with another method yeah? Just not as easy?

**Reply by itzg** [reply]
Correct. I believe this approach still works for FTB modpacks, since it was first implemented for theirs

https://github.com/itzg/docker-minecraft-server#running-a-server-with-a-curseforge-modpack

**Comment by mattdood** [comment]
If you'd like to continue using the `TYPE=FTBA` you can find the modpack ID # by mousing over one of the server files download links on the website. I ran into this issue, but noticed that the site uses this format for the download: 
`https://api.modpacks.ch/public/modpack/<id>/<download version>/server/linux`

**Reply by davidfifth** [reply]
Good catch! Thanks for sharing
